quoridor-app/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
+    "eject": "expo eject"
+  },
+  "dependencies": {
+    "expo": "^33.0.0",
+    "react": "16.8.3",
```

Comment:
Why is the react version fixed, unlike other dependencies?

Reply:
expo init did it. I change them in my next feature

Reply:
What is done is no longer to be done ^^